src/main/java/uk/gov/hmcts/reform/judicialapi/configuration/MethodSecurityConfiguration.java

```diff
@@ -0,0 +1,31 @@
+package uk.gov.hmcts.reform.judicialapi.configuration;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.access.AccessDecisionManager;
+import org.springframework.security.access.AccessDecisionVoter;
+import org.springframework.security.access.expression.method.ExpressionBasedPreInvocationAdvice;
+import org.springframework.security.access.vote.AffirmativeBased;
+import org.springframework.security.access.vote.AuthenticatedVoter;
+import org.springframework.security.access.vote.RoleVoter;
+import org.springframework.security.config.annotation.method.configuration.EnableGlobalMethodSecurity;
+import org.springframework.security.config.annotation.method.configuration.GlobalMethodSecurityConfiguration;
+
+
+@Configuration
+@EnableGlobalMethodSecurity(securedEnabled = true)
+public class MethodSecurityConfiguration extends GlobalMethodSecurityConfiguration {
+

```

Comment:
Any use of this class. Not sure why we need

Reply:
We need this class for the validation of the roles when calling the api, without it the security validation doesn't work.